core/src/stored/btape.cc

```diff
@@ -208,9 +208,9 @@ int main(int margc, char* margv[])
 
   uint32_t x32 = 123456789;
```

Comment:
I feel like these tests should be part of our test suite and not part of btape.

Why does btape care for scanf/printf anyways ?

Reply:
bscan is the only user of `bsscanf()` and that seems to have had a bug some time in the past...

Reply:
ok... I just realized we have `#define sscanf bsscanf` in baconfig.h. We also have `#define fgets bfgets` so we have the "safer" home-grown functions for that...
I'd say we leave it like that for now and remove `bsscanf()` and `bfgets()` as soon as we have proper tests for those.

Reply:
I already tried once to switch to sscanf instead of bscanf.  Sadly sscanf handles some things differently than bsscanf.  Iirc the biggest problem was that sscanf does not match empty strings, e.g.
```
sscanf("volume=%127s other=%d\n", "volume= other=5\n", &vol, &other) == 0
```
but
```
bsscanf("volume=%127s other=%d\n", "volume= other=5\n", &vol, &other) == 2
```
i.e. our message parsing will break with that change.  Im not sure how much effort it would be to fix this, especially if we want forwards/backwards compatibilty.